### PR TITLE
Update Flutter SDK

### PR DESCRIPTION
**Pull request by jumaallan**
Story: N/A

## Summary

Updates the Flutter SDK with new changes from native SDKs

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+# 10.0.0-beta03
+- Allow setEnvironment({required bool useSandbox}) to enable sandbox or production environment
+- Allow setCallbackUrl({required Uri callbackUrl}) to set a callback url for all submitted jobs.
+- Bug Fixes and Improvements from iOS v10.0.0-beta10 and Android v10.0.0-beta08
```
Comment (vanshg) [inline CHANGELOG.md:6]:
```suggestion
- Bug Fixes and Improvements from iOS v10.0.0-beta10 and Android v10.0.0-beta09
```

File: android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt
Comment (vanshg) [inline android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt:1]: I thought we'd ignored generated files from ktlint?

Comment (jumaallan) [reply android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt:1]: Yes, does it show still?

Let me check

Comment (jumaallan) [reply android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt:1]: Seems to pass all checks

Including lint checks

Comment (vanshg) [reply android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt:1]: Oh were these changes from running Pigeon normally? the pigeon version updates caused some changes to the already generated code?

**Review by vanshg — APPROVED** [review]

File: android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt
Comment (jumaallan) [reply android/src/main/kotlin/com/smileidentity/flutter/generated/Messages.g.kt:1]: Will look into that
